xenon/__init__.py

```diff
@@ -36,6 +36,9 @@ def parse_args():
                         help='Comma separated list of patterns to ignore. If '
                         'they are directories, Xenon won\'t even descend into '
                         'them')
+    parser.add_argument('-ib', '--ignore-blocks', metavar='<str>', dest='ignore_blocks',
```

Comment:
I'd like to keep the convention that short options have a single letter only. Let's pick a letter that is currently unused.